flake.nix

```diff
@@ -20,11 +20,8 @@
     devour-flake.url = "github:srid/devour-flake";
     devour-flake.flake = false;
 
-    # TODO: Use upstream after https://github.com/NixOS/nix/pull/8892
-    # Note: This version of nix is only used to run `nix flake show` in omnix-cli
-    # Also note: Using shivaraj-bh fork of nix which fixes x86_64-darwin on top of github:DeterminateSystems/nix-src/flake-schemas
-    nix.url = "github:shivaraj-bh/nix/flake-schemas";
-    nix.inputs.flake-parts.follows = "flake-parts";
+    inspect.url = "github:juspay/inspect";
```

Comment:
I recommend not pushing to `main` branch on forks.

Reply:
Also, let's open a PR for this change. Not sure if 1 PR or two PRs, since https://github.com/juspay/inspect/commit/279cfa5c434e5bbebb4c07fa6af2deeb84e03bb0 refers to two things.

Reply:
https://github.com/juspay/omnix/pull/277/commits/2ab059c8ff56596914247485cd711d8588870f5a

Reply:
> Also, let's open a PR for this change. Not sure if 1 PR or two PRs, since [juspay/inspect@279cfa5](https://github.com/juspay/inspect/commit/279cfa5c434e5bbebb4c07fa6af2deeb84e03bb0) refers to two things.

Rationale: we want to move away from the culture of maintaining monolithic fork branches for long time. 

Ideally there would be two PRs upstream -- one for specifying flake-schemas as input, and another for adding `evalOnAllSystems` (I imagine the 2nd one will see some discussion among maintainers).

Then, we can have one branch in our fork, where commits from these two PRs are merged - and use it here (until the PRs get merged upstream).

Reply:
PR for flake-schemas input: https://github.com/DeterminateSystems/inspect/pull/6

Discussion about `evalOnAllSystems`: https://github.com/DeterminateSystems/inspect/issues/7